specs/operator/rbac.spec

```diff
@@ -0,0 +1,16 @@
+# Verify RBAC Resources
+
+## Disable RBAC resource creation:
+Tags: rbac-disable
+Component: Operator
+Level: Integration
+Type: Functional
+Importance: High
+
+This scenario helps you to disable creation of RBAC resources at cluster level.
```

Comment:
Test case ID is missing

Reply:
What is Test case ID ?

Reply:
use `PIPELINES-11-TC01` Ref: https://github.com/openshift-pipelines/release-tests/blob/83d34d5c21c4a4036c21650cf4f1b9be5a0b8e7a/specs/triggers/eventlistener.spec#L298